stories/useFriction.stories.tsx

```diff
@@ -3,49 +3,94 @@ import { withKnobs, number } from '@storybook/addon-knobs';
 
 import { useFriction } from '../src';
 import './index.css';
+import './toggle.css';
+import './button.css';
 
 export default {
   title: 'Friction',
   decorators: [withKnobs],
 };
 
-export const FrictionOpacity: React.FC = () => {
+export const FrictionMultiple: React.FC = () => {
+  const [toggle, setToggle] = React.useState(true);
+
   const [props] = useFriction<HTMLDivElement>({
-    from: { opacity: 0 },
-    to: { opacity: 1 },
+    from: {
+      opacity: toggle ? 0 : 1,
+      transform: toggle
+        ? 'scale(0) rotate(0deg) translate(-50%, -50%)'
+        : 'scale(1) rotate(180deg) translate(-50%, -50%)',
+    },
+    to: {
+      opacity: toggle ? 1 : 0,
+      transform: toggle
+        ? 'scale(1) rotate(180deg) translate(-50%, -50%)'
+        : 'scale(0) rotate(0deg) translate(-50%, -50%)',
+    },
     config: {
       mu: number('mu', 0.25),
       mass: number('mass', 50),
-      initialVelocity: number('initialVelocity', 5000),
+      initialVelocity: number('initialVelocity', 5),
     },
   });
 
-  return <div className="mover mover--opacity" {...props} />;
+  return (
+    <>
+      <div className="toggle">
+        <input
+          type="checkbox"
+          name="toggle"
+          id="toggle"
+          checked={toggle}
+          onChange={() => {
+            setToggle(prevToggle => !prevToggle);
+          }}
+        />
+        <label htmlFor="toggle"></label>
+      </div>
+      <div className="mover mover--opacity" {...props} />
+    </>
+  );
 };
 
 export const FrictionTranslateX: React.FC = () => {
-  const [props] = useFriction<HTMLDivElement>({
+  const [props, controller] = useFriction<HTMLDivElement>({
     from: { transform: 'translateX(0px) translate(-50%, -50%)' },
     to: { transform: 'translateX(300px) translate(-50%, -50%)' },
     config: {
-      mu: number('mu', 0.05),
+      mu: number('mu', 0.5),
       mass: number('mass', 300),
-      initialVelocity: number('velocity', 1000),
+      initialVelocity: number('velocity', 10),
     },
+    immediate: false,
   });
 
-  return <div className="mover mover--translate" {...props} />;
+  return (
+    <>
+      <button className="button" onClick={controller.start}>
```

Comment:
The first use of `controller` – wiring it to a button click. The animation won't run until `controller.start` is called. Notice the `immediate: false` above. I'd love any thoughts on this API and whether or not it feels right // intuitive.

Reply:
Yea, I think this totally works. 

However, I feel like a lot of it feeling a little funky is the fact that if you think about these transitions in terms of a finite automata, and you have `from` and `to`, It's almost as if your describing a the transition in terms of the current vertex as opposed to the possible state nodes it could be at and letting your physics handle the transition between the nodes.

I think another way of looking at this is you have two states `stateOne` & `stateTwo`. what you wanna do `onClick` is transition the current state _from_ `stateOne` _to_ `stateTwo` and let the `Friction` handle that interpolation. 

With the current API, you basically have to keep a mental model of the whole automata `stateOne <-> stateTwo`. you then describe a vertex with `from` & `to` (in this example you're describing the vertex: `stateOne -> stateTwo`). Then it becomes an implicit state transition by saying "execute the vertex" as opposed to an explicit state transition of "update to the new state".

I think if the api where altered the logical equivalent of what you have here wouldn't look as funky:
```
const [props, controller] = useFriction<HTMLDivElement>({
    states: {
      one: { transform: 'translateX(0px) translate(-50%, -50%)' },
      two: { transform: 'translateX(300px) translate(-50%, -50%)' },
    }
    initialState: 'one'
    config: {
      mu: number('mu', 0.05),
      mu: number('mu', 0.5),
      mass: number('mass', 300),
      initialVelocity: number('velocity', 1000),
      initialVelocity: number('velocity', 10),
    }
  });
...
   <button className="button" onClick={() => controller.setState('two')}>
```

I'm not saying you have to change your API by any means, but just saying that this is what's going on and how to reason about what's logically happening under the hood with the current API design. At least I think 🤷‍♂

Hopefully this helps. let me know if I'm completely off the mark and misunderstanding something too

Reply:
I love this description Max! It actually is really helpful to think about these animations in terms of states. In that sense, `initialState` of something like `init` or `standby` could be a really nice way to convey that the animation isn't yet running, but is _ready_ to run when executed. I don't think I'll necessarily be able to get to this by the time the Fellowship ends, but this could be a really excellent API, especially for things like sequencing animations. Thanks for the explanation, I'll make a longer term issue for this so we can track this alternative API design, because I think it's really cool and I would like to at least explore it.

Reply:
Dope. Yea, I don't think there's anything wrong with the current api describing the expected vertex to execute, but just another lens to view it through that could help with naming and other api decisions built on top of the current abstraction in the future.

Keep up the good work man. you're killing it